rolecall/rolecall.py

```diff
@@ -7,92 +7,37 @@
 import logging
 import os
 from copy import deepcopy
+import json
+import threading
```

Comment:
unused import

Reply:
removed unused import of threading as of commit 6d7ebd4